src/commands/init.ts

```diff
@@ -47,72 +46,34 @@ import { getArtifactsPath } from "../lib/pathsGetters/getArtifactsPath";
 import { getSrcAquaDirPath } from "../lib/pathsGetters/getSrcAquaDirPath";
 import { input } from "../lib/prompt";
 
+export const NAME_OR_PATH = "NAME-OR-PATH";
+
 export default class Init extends Command {
-  static override description =
-    "Initialize fluence project in the current directory";
+  static override description = "Initialize fluence project";
   static override examples = ["<%= config.bin %> <%= command.id %>"];
-  // TODO DXJ-31: add "--path" optional flag to set path of the project
   static override flags = {
     ...NO_INPUT_FLAG,
   };
-  static override args = [{ name: NAME_ARG, description: "Project name" }];
+  static override args = [
+    {
+      name: NAME_OR_PATH,
+      description: "New project directory name or path",
+    },
+  ];
   static override usage: string = usage(this);
   async run(): Promise<void> {
     const { args, flags } = await this.parse(Init);
     const isInteractive = getIsInteractive(flags);
-    const projectName: unknown = args[NAME_ARG];
-    assert(projectName === undefined || typeof projectName === "string");
-    await init(this, isInteractive, projectName);
+    const nameOrPath: unknown = args[NAME_OR_PATH];
+    assert(nameOrPath === undefined || typeof nameOrPath === "string");
```

Comment:
I was wondering if it's possible to force parsed args (`args` variable) to have the right type?